### PR TITLE
Website demo

### PR DESCRIPTION
**Pull request by runsenwu**
Added CSS and html changes for the new UI, also integrate the video to the UI well.

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (sangwo) [inline .idea/vcs.xml]: I believe `.idea` folder contains intellij settings specific to you & your ide. Maybe add `.idea` to `.gitignore`?

File: .idea/workspace.xml [hunk 2]
```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (sangwo) [inline .idea/workspace.xml]: Same for this

File: static/lib/style.css [hunk 3]
```diff
@@ -0,0 +1,514 @@
...
+}
+
+:root {
+    --color-main: #142160;
+    --bg: #fbfefd;
+    --bg2: #dce5ff6c;
+    --main-accent: #e9eefd;
+    --main-text: #4b5976;
+    --shadow: rgba(17, 17, 26, 0.1) 0px 0px 16px;
+
```
Comment (sangwo) [inline static/lib/style.css:19]: Could you remove extra empty line? Just to keep the codebase clean! Idk if it exists in intellij (I use vscode), but I recommend installing ide extension such as "Trailing Spaces" that shows you trailing whitespaces.

File: static/lib/style.css [hunk 4]
```diff
@@ -0,0 +1,514 @@
...
+}
+
```
Comment (sangwo) [inline static/lib/style.css:21]: Same here; please check the rest of the file as well; I see some inconsistent extra empty line / no new line between styles (unless you intended)

File: templates/index.html [hunk 5]
```diff
@@ -1,98 +1,272 @@
...
-			height: 375px;
-			background-color: #666;
-		}
-		#canvas {
-			display: hidden
-		}
-	</style>
+<html lang="en">
+	<head>
+		<script src="static/lib/jquery-3.6.0.min.js"></script>
```
Comment (sangwo) [inline templates/index.html]: Indentation is inconsistent throughout the file; please fix them!

File: templates/style.css [hunk 6]
```diff
@@ -0,0 +1,514 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap');
```
Comment (sangwo) [inline templates/style.css:1]: Is this file redundant with `static/lib/style.css`? Please consider removing one

File: templates/index.html [hunk 7]
```diff
@@ -1,98 +1,272 @@
...
+		<meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"/>
+    <link rel="stylesheet" href="static/lib/style.css">
```
Comment (sangwo) [inline templates/index.html]: Please only keep one of the `style.css` and remove link to the other file.

File: templates/index.html [hunk 8]
```diff
@@ -1,98 +1,272 @@
...
-			} else {
-				clearPhoto();
-			}
-		}
-	</script>
+    <div class="sidebar">
+        <div class="sidebar-container">
+         <div class="brand">
+             <h3>
+                 <!-- <span class="fa-brands fa-staylinked"></span> -->
```
Comment (sangwo) [inline templates/index.html]: Please remove commented out code; please check the rest of the file as well

File: templates/index.html [hunk 9]
```diff
@@ -1,98 +1,272 @@
...
+              </button>
+
+         </div> -->
+
+        </div>
+    </div>
+
+
+    <div class="main-content">
+
```
Comment (sangwo) [inline templates/index.html]: Please remove unnecessary new lines (unless they are intended; eg. to separate different parts/components)

File: website_demo/CNAME [hunk 10]
```diff
@@ -0,0 +1 @@
+wedding.rampatra.com
```
Comment (sangwo) [inline website_demo/CNAME]: Is this file necessary? Please consider removing

File: website_demo/README.md [hunk 11]
```diff
@@ -0,0 +1,39 @@
+# Wedding Website
```
Comment (sangwo) [inline website_demo/README.md]: Please remove this file and images that aren't relevant to our website; there's A LOT of them

File: website_demo/npmfile.js [hunk 12]
```diff
@@ -0,0 +1,3 @@
+exports.printMsg = function() {
+    console.log("Visit https://wedding.rampatra.com !");
```
Comment (sangwo) [inline website_demo/npmfile.js]: What does this do? Please consider removing or updating

File: website_demo/browserconfig.xml [hunk 13]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig>
+    <msapplication>
+        <tile>
+            <square150x150logo src="/mstile-150x150.png"/>
```
Comment (sangwo) [inline website_demo/browserconfig.xml]: What does this file do? I think `mstile-150x150.png` is not relevant to our website. Please consider removing or updating

**Review by sangwo — CHANGES_REQUESTED** [review]
Thanks for your work! I made some comments on specific files (regarding style & unnecessary files). Please review & update

File: templates/style.css (hunk 6 above)
Comment (runsenwu) [reply templates/style.css:1]: One of the CSS file is for testing (the one in templates) and the other CSS file (the one in static) is for when running flask. For some reason for me to open and test in default browser, I have to have the CSS within the local folder. And for flask it only registers the file in the static folder.

File: templates/index.html (hunk 7 above)
Comment (runsenwu) [reply templates/index.html]: One of the CSS file is for testing (the one in templates) and the other CSS file (the one in static) is for when running flask. For some reason for me to open and test in default browser, I have to have the CSS within the local folder. And for flask it only registers the file in the static folder.

File: templates/index.html [hunk 14]
```diff
@@ -1,98 +1,229 @@
...
-		<video autoplay="true" id="video"></video>
-		<button id="capture">Capture</button>
-	</div>
+    <head>
+        <script src="static/lib/jquery-3.6.0.min.js"></script>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css" />
+        <link rel="stylesheet" href="static/lib/style.css">
```
Comment (sangwo) [inline templates/index.html:10]: I think you might be using the wrong path. Try using `../static/lib/style.css` instead, remove the other link, and see if that works? If it works, please remove the redundant css file (one in templates folder).

File: website_demo/index.html [hunk 15]
```diff
@@ -0,0 +1,326 @@
...
+                        <div class="logo"><a href="#"><img src="img/hcp_logo_64.png" alt="Logo"></a></div>
+                        <div class="header-nav">
+                            <nav>
+                                <ul class="primary-nav">
+                                    <li><a href="#intro">About Us</a></li>
+                                    <li><a href="#events">Demo</a></li>
+                                    <li><a href="#instagram"></a></li>
+                                </ul>
+                                <ul class="member-actions">
+                                    <!-- <li><a href="#map" class="login">Venue</a></li> -->
```
Comment (sangwo) [inline website_demo/index.html]: There are some commented out code that you forgot to remove! Please check the rest of the file as well.

File: website_demo/index.html [hunk 16]
```diff
@@ -0,0 +1,326 @@
...
+                    <div class="col-sm-4 hidden-xs"></div>
+                    <div class="col-sm-4 hidden-xs text-center"><a href="#rsvp"
+                            class="btn btn-accent btn-large rsvp-btn">Join our mailing list</a></div>
+                    <div class="col-sm-4 hidden-xs"></div>
+                </div>
+            </div>
+            <div class="down-arrow floating-arrow"><a href="#invitation"><i class="fa fa-angle-down"></i></a></div>
+        </section>
+
+        <section id="invitation" class="section-padding">
```
Comment (sangwo) [inline website_demo/index.html]: Could you remove unused sections? I believe `invitation`, `instagram`, `eng-pics`, `video-bg` are unused. We can add back / create new sections when needed.

File: website_demo/index.html [hunk 17]
```diff
@@ -0,0 +1,326 @@
...
+                    <div class="col-md-12">
+                        <h3 class="header">Demo</h3>
+                    </div>
+                </div>
+                <div>
+                    <A HREF="../templates/index.html">About this site</A>
+                </div>
+            </div>
+
+            <div id="dc-modal" class="modal fade" tabindex="-1" role="dialog">
```
Comment (sangwo) [inline website_demo/index.html]: Could you remove this modal (entire div)? This doesn't seem to appear & relevant to our project.

**Review by sangwo — CHANGES_REQUESTED** [review]
Thanks for the quick response! I've added a few more comments :)

File: templates/index.html (hunk 14 above)
Comment (runsenwu) [reply templates/index.html:10]: Okay, just tried it again to double check, it does not work unfortunately : (

We will remove it once we have the UI completely in place

File: website_demo/index.html (hunk 16 above)
Comment (runsenwu) [reply website_demo/index.html]: I was thinking about leaving Instagram to leave pictures of our product and potentially use video-bg as an add-on. But I removed the other two that were not being used

**Review by sangwo — APPROVED** [review]
